LongLookupTables/get_long_lookup_tables.py

```diff
@@ -0,0 +1,136 @@
+# import tqdm
+import os
+import json
+
+from LongLookupTables.make_long_lookup_tables import make_long_lookup_tables
+from utils.get_default_params import get_default_params
+
+dir_path = os.path.dirname(os.path.realpath(__file__))
+
+# Dictionary that maps input name of lookup table dataset
+# to directory where it is located
+name2dir = {
+    # Lookup tables with training up to 3 compositions
+    "long_lookup": "LongLookupTables",
+
+    # Lookup tables with training 1 2 and 4 compositions (i.e jumping 3)
+    # - currently does not have appropriate generation.txt
+    # "long lookup jump": "LongLookupTablesJump",
+
+    # long lookup tables with a iniital training file without t7 and t8
+    # and then adding uncomposed t7 and t8 with all the rest
+    "long_lookup_oneshot": "LongLookupTablesOneShot",
+
+    # reverse long lookup table (i.e right to left hard attention)
+    "long_lookup_reverse": "LongLookupTablesReverse",
+
+    # noisy long lookup table where between each "real" table there's one noisy
+    # one. THe hard attention is thus a diagonal wich is less steep
+    "long_lookup_intermediate_noise": "LongLookupTablesIntermediateNoise",
+
+    # noisy long lookup table with a special start token saying when are the
+    # "real tables" starting. THe hard attention is thus a diagonal that
+    # starts at some random position.
+    "noisy_long_lookup_single": "NoisyLongLookupTablesSingle",
+
+    # noisy long lookup table where there are multiple start token and only
+    # the last one really counts
+    "noisy_long_lookup_multi": "NoisyLongLookupTablesMulti"
+}
+
+
+def get_long_lookup_tables(name, is_small=False, is_mini=False, longer_repeat=5):
+    """
+    Return the wanted lookup dataset information, downloads or generates
+    it if it is not already present
+
+    Args:
+        name ({"long_lookup", "long_lookup_oneshot", "long_lookup_reverse", 
+            "noisy_long_lookup_multi", "noisy_long_lookup_single",
```

Comment:
All the comments in `name2dir` about the datasets should be in the docstrings. I know that's not how it's done in my code. But clean code should (besides special / complex cases) not have any comments and just use docstrings and functions to make things clear.  Importantly all information that the user would be interested in should be in the docstrings. The explanation of the datasets is definitely such a case.
Please change that : put them under the argument Name or under a Note

Reply:
Done, also added all these comments in get_tasks (PR 3), since they are useful to anyone unsure of how datasets differ.